app/assets/javascripts/settings.js

```diff
@@ -20,7 +20,7 @@ $(document).on('turbolinks:load', function(){
   var action = $("body").data('action');
 
   // Only run on the settings page.
-  if ((controller == "users" && action == "edit") || (controller == "users" && action == "update")){
+  if ((controller == "users" && action == "edit") || (controller == "users" && action == "update") || (controller == "admins" && action == "index")){
```

Comment:
Was this fixed somewhere else? I have the idea that you had found a better way of filtering this

Reply:
As far as I know, this is the simplest & cleanest way to prevent js from running on every page. In this case the code is very specific to the settings pages so allowing it to work on any page would not be ideal